A test is erroneous if it raises an exception (without catching it)
